File: app/src/main/java/com/computop/android/sdk/example/MainActivity.java
package com.computop.android.sdk.example;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.support.v7.widget.Toolbar;

import butterknife.BindView;
import butterknife.ButterKnife;
import butterknife.Unbinder;

public class MainActivity extends AppCompatActivity {

    @BindView(R.id.toolbar)
    Toolbar toolbar;

    private static final String TAG = MainActivity.class.getSimpleName();
    private Unbinder unbinder;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        unbinder = ButterKnife.bind(this);

        showLandingPage();
    }

    private void showLandingPage() {
        toolbar.setTitle(getString(R.string.app_name));

        getSupportFragmentManager()
                .beginTransaction()
                .replace(R.id.fragment_container, new LandingPage(), LandingPage.class.getCanonicalName())
                .commit();
    }

    @Override
    protected void onDestroy() {
        super.onDestroy();
        unbinder.unbind();
    }
}

File: app/src/main/java/com/computop/android/sdk/example/MainActivity.kt
package com.computop.android.sdk.example

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.computop.android.sdk.example.databinding.ActivityMainBinding
import com.google.android.material.appbar.CollapsingToolbarLayout

class MainActivity : AppCompatActivity() {

    private var _binding: ActivityMainBinding? = null
    private val binding
        get() = _binding!!

    private var collapsingToolbarLayout: CollapsingToolbarLayout? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        _binding = ActivityMainBinding.inflate(layoutInflater)

        val view = binding.root
        setContentView(view)

        collapsingToolbarLayout = binding.mainCollapsing
    }

    override fun onStart() {
        super.onStart()

        showLandingPage()
    }

    private fun showLandingPage() {
        binding.toolbar.title = getString(R.string.app_name)

        supportFragmentManager
            .beginTransaction()
            .replace(R.id.fragment_container, LandingPage(), LandingPage::class.java.canonicalName)
            .commit()
    }

    override fun onDestroy() {
        super.onDestroy()
        _binding = null
    }
}